src/pybind/mgr/cephadm/serve.py

```diff
@@ -475,6 +475,9 @@ def matches_network(host):
         remove_daemon_hosts: Set[orchestrator.DaemonDescription] = ha.remove_daemon_hosts(hosts)
         self.log.debug('Hosts that will loose daemons: %s' % remove_daemon_hosts)
 
+        if daemon_type == 'HA_RGW':
```

Comment:
```suggestion
        if daemon_type == 'ha-rgw':
```

let's keep things lowercase.